src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: User | null;
```

Comment:
You can also use '?' instead of null. If you implement it you can also chenge the conditions in other files by removing  '  | |  null ' 
```suggestion
  user?: User;
```

Reply:
> You can also use '?' instead of null. If you implement it you can also chenge the conditions in other files by removing ' | | null '

It's not necessary